sensor_msgs/Image c_image
visualization_msgs/Marker c_marker